### PR TITLE
Subscription form

### PR DESCRIPTION
**Pull request by CharlyJuteau**

### CODE REVIEW

**Review by cartige — DISMISSED** [review]

File: frontend/src/components/Modal/modal.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.modalOverlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: 1040;
```
Comment (LoicBrassart) [inline frontend/src/components/Modal/modal.scss]: You don't understand z-index?I suppose ? -_-

File: frontend/src/components/Modal/modal.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.modalWrapper {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1050;
```
Comment (LoicBrassart) [inline frontend/src/components/Modal/modal.scss]: You don't understand z-index?I suppose ? -_-

File: frontend/src/components/Modal/modal.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  height: 100%;
+  overflow-x: hidden;
+  overflow-y: auto;
+  outline: 0;
+  display: flex;
+  align-items: center;
+}
+
+.modal {
+  z-index: 100;
```
Comment (LoicBrassart) [inline frontend/src/components/Modal/modal.scss]: You don't understand z-index?I suppose ? -_-

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Look this video, and fix your z-indexes please

https://youtu.be/uS8l4YRXbaw

**Review by cartige — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Better

**Pull request merged**